src/itemset.js

```diff
@@ -115,11 +115,13 @@ define([
                     var possibleSrcs = _.map(datasources.getPossibleFixtures(), 
                                              function(val) { return val.src; }),
                         notCustom = _.contains(possibleSrcs, itemsetData.instance.src),
-                        choices = datasources.autocompleteChoices(itemsetData.instance.src);
+                        choices = datasources.autocompleteChoices(itemsetData.instance.src),
+                        strippedValue = itemsetData.valueRef.replace(/(\[.+])+/, ""),
+                        strippedLabel = itemsetData.labelRef.replace(/(\[.+])+/, "");
```

Comment:
To a future dev reading this code, though, that's going to look like a possible bug (that `instance("foo")/foo/bar[a=1]/baz[b=2]` becomes `instance("foo")/foo/bar`). How about `/(\[^\]+])+/` to make it more obvious that you only want to replace filters?